tests/unit_tests/movies/HelioviewerMovieTest.php

```diff
@@ -0,0 +1,187 @@
+<?php declare(strict_types=1);
+/**
+ * @author Daniel Garcia-Briseno <daniel.garciabriseno@nasa.gov>
+ */
+
+use PHPUnit\Framework\TestCase;
+use Helioviewer\Api\Event\EventsStateManager;
+
+// Dependencies
+include_once HV_ROOT_DIR.'/../src/Helper/RegionOfInterest.php';
+include_once HV_ROOT_DIR.'/../src/Database/MovieDatabase.php';
+include_once HV_ROOT_DIR.'/../src/Movie/HelioviewerMovie.php';
+
+final class HelioviewerMovieTest extends TestCase
+{
+    const TEST_LAYER = "[SOHO,LASCO,C2,white-light,2,100,0,60,1,2024-07-11T09:03:05.000Z]";
+
+    /**
+     * Inserts a test movie into the database and returns its public ID
+     */
+    static private function InsertSohoTestMovie(): string {
+        // Get a handle to the movie database
+        $movieDb = new Database_MovieDatabase();
+        // Set up arguments needed to insert a movie into the database
+        $imageScale = 16;
+        $roi = new Helper_RegionOfInterest(-16600, -8000, 16600, 8000, $imageScale);
+        $soho_layer = HelioviewerMovieTest::TEST_LAYER;
+        $layers = new Helper_HelioviewerLayers($soho_layer);
+        $events_manager = EventsStateManager::buildFromEventsState([]);
+        // Insert a test movie
+        $dbId = $movieDb->insertMovie(
+            "2023-12-01 00:00:00",
+            "2023-12-01 01:00:00",
+            "2023-12-01 00:30:00",
+            $imageScale,
+            $roi->getPolygonString(),
+            10,
+            true,
+            "[SOHO,LASCO,C2,white-light,2,100,0,60,1,2024-07-11T09:03:05.000Z]",
```

Comment:
Use TEST_LAYER

Reply:
also try with / instead of -

Reply:
/ and - don't matter to the API since they're passed to mysql and mysql does the right thing...